doc_generator/utility/export.py

```diff
@@ -0,0 +1,50 @@
+# SPDX-FileCopyrightText: 2023 Andreas Kahler <mail@andreaskahler.com>
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import cadquery as cq
+import cadscript
+
+def export_svg(part, filename):
+    '''
+    Export a part to SVG.
+    '''
+
+    if isinstance(part, cadscript.Body):
+      part = part.cq()
+
+    cq.exporters.export(part,
+                        str(filename),
+                        opt={
```

Comment:
We are probably going to need control over these options at some point, but it is fine as-is for now to get us up and running.